Chaincase.iOS/Services/iOSTorManager.cs

```diff
@@ -290,5 +292,41 @@ private void CancelInitRetry()
             initRetry?.Cancel();
             initRetry = null;
         }
+
```

Comment:
The next release of github.com/iCepa/Tor.framework will include official support for hosting v3 hidden services

https://github.com/iCepa/Tor.framework/issues/33#issuecomment-1046838630